addon/components/bootstrap-inputs/-text.js

```diff
@@ -33,6 +23,7 @@ export const propDefinitions = {
     type: PropTypes.string,
   },
   label: {
+    default: 'Text',
     description: 'The label for the input.',
     type: PropTypes.string,
```

Comment:
Again this label should also be required, might as well patch the others too that were not build in this PR.

Reply:
I don't think a vanilla input should have a default label though. I can't think of any application where I would want an input with the label `Text`

Reply:
I would agree with @Samsinite.